ocs_ci/ocs/constants.py

```diff
@@ -962,3 +963,7 @@
 
 # cluster expansion
 MAX_OSDS = 15
+
+# Minimum cluster requirements in term of node specs
+MIN_NODE_CPU = 16
```

Comment:
Maybe we should get it from a yaml file as we will also have lower speced clusters

Reply:
this value is for the product's required minimum spec